src/metaswitch/clearwater/etcd_shared/common_etcd_synchronizer.py

```diff
@@ -131,6 +131,7 @@ def __init__(self, plugin, ip, etcd_ip=None):
         self._index = None
         self._last_value = None
         self._terminate_flag = False
+        self._abort_read = False
```

Comment:
Worth a comment to explain this flag? (It might not be obvious what the difference from _terminate_flag is, or when this gets set.)